src/include/duckdb/common/unique_ptr.hpp

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include "duckdb/common/unique_ptr_utils.hpp"
+
+#include <memory>
+#include <type_traits>
+
+using std::add_lvalue_reference;
+using std::default_delete;
```

Comment:
- ```suggestion

```
I would avoid pollution stuff in headers (since it's then pulled in by anyone)